client/src/templates/Introduction/components/super-block-accordion.tsx

```diff
@@ -210,8 +174,61 @@ export const SuperBlockAccordion = ({
   chosenBlock,
   completedChallengeIds
 }: SuperBlockAccordionProps) => {
+  const getCorrectStructure = (superBlock: SuperBlocks) => {
```

Comment:
The Full Stack Cert structure was hard coded here, so I needed to make it dynamic. Now it will choose FullStackOpen or FullStackCert structure depending on the course.